packages/fxa-admin-panel/package.json

```diff
@@ -35,8 +35,7 @@
     ]
   },
   "dependencies": {
-    "apollo-boost": "^0.4.9",
-    "apollo-client": "^2.6.4",
+    "@apollo/client": "^3.2.1",
```

Comment:
Is this the same exact version used everywhere else?

Reply:
Oh looks like Settings uses `3.1.3`, I could upgrade that while I'm at it.